requirements.txt

```diff
@@ -0,0 +1,3 @@
+Jinja2
```

Comment:
It's helpful to have a requirements file if we have to run these scripts locally.